orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/TerminateInstancesTask.groovy

```diff
@@ -38,13 +40,23 @@ class TerminateInstancesTask extends AbstractCloudProviderAwareTask implements T
   @Autowired
   TerminatingInstanceSupport instanceSupport
 
+  @Autowired
+  TrafficGuard trafficGuard
+
   @Override
   TaskResult execute(Stage stage) {
     String cloudProvider = getCloudProvider(stage)
     String account = getCredentials(stage)
 
     List<TerminatingInstance> remainingInstances = instanceSupport.remainingInstances(stage)
 
+    trafficGuard.verifyInstanceTermination(
+      stage.context.instanceIds as List<String>,
+      account,
+      Location.region(stage.context.region as String),
```

Comment:
We only ever send across `stage.context.region` right?

Not sure about this stage but historically we had the whole `region` vs `regions` thing and had support for both sprinkled throughout.

Just a question, not sure it applies here.